README.md

```diff
@@ -1,3 +1,15 @@
 # Download service
 
-Backend server for [https://github.com/cta-epfl/ctadata](https://github.com/cta-epfl/ctadata).
\ No newline at end of file
+Backend server for [https://github.com/cta-epfl/ctadata](https://github.com/cta-epfl/ctadata).
+
+## Installation
+
+For `production` use to install dependencies just run :
```

Comment:
it's not certain that production use of this service will be always within jupyterhub.

```suggestion
For use as a JupyterHub service, run:
```